src/extensions/http_api/definitions.js

```diff
@@ -184,6 +191,13 @@ exports.install = ({ baseUrl = '' } = {}) => {
         console.log(inspect(response.body, { colors: true, depth: null })) // eslint-disable-line no-console
     })
 
+    /**
+     * Dumping state
+     */
+    When(/^(?:I )?dump state$/, function () {
```

Comment:
This already exists, see https://github.com/ekino/veggies#state-gherkin-expressions and https://github.com/ekino/veggies/blob/master/src/extensions/state/definitions.js#L15

Reply:
Thank you Yohan, I don't pay attention to that.